### PR TITLE
Load flow provider selection

### PR DESCRIPTION
**Pull request by geofjamg**
Signed-off-by: Geoffroy Jamgotchian <geoffroy.jamgotchian@rte-france.com>

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -217,5 +217,12 @@
...
     "ColumnsList": "Column list",
     "LabelSelectList": "Edit columns",
-    "CheckAll": "Select all / none"
+    "CheckAll": "Select all / none",
+
+    "Provider": "Provider: ",
+    "Default": "Default",
+    "OpenLoadFlow": "Open Load Flow",
+    "Hades2": "Hades 2",
+    "setLoadFlowProviderError": "An error occured when updating the provider",
```
Comment (FranckLecuyer) [inline src/translations/en.json:226]: An error occured when updating the load flow provider

File: src/components/parameters.js [hunk 2]
```diff
@@ -635,7 +696,7 @@ const Parameters = ({ showParameters, hideParameters }) => {
                         <MapParameters />
                     </TabPanel>
                     <TabPanel value={tabIndex} index={2}>
-                        {studyUuid && <LoadFlowParameters />}
+                        {studyUuid && <LoadFlow />}
                     </TabPanel>
                     <Grid item xs={12}>
                         <Button
```
Comment (FranckLecuyer) [inline src/components/parameters.js:702]: After a modification in the load flow provider, the loadflow run button does not change (it remains green and we cannot rerun a loadflow, without doing a modification in the study to rerun it again)
See my remark in study-server : https://github.com/gridsuite/study-server/pull/91/files?file-filters%5B%5D=.java&file-filters%5B%5D=.sql&hide-deleted-files=true

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/translations/en.json (hunk 1 above)
Comment (geofjamg) [reply src/translations/en.json]: Fixed

File: src/components/parameters.js (hunk 2 above)
Comment (geofjamg) [reply src/components/parameters.js:702]: Fixed in study-server

**Review by FranckLecuyer — APPROVED** [review]

**Pull request merged**